gsa/src/web/pages/scanconfigs/editdialog.js

```diff
@@ -79,13 +79,13 @@ class NvtPreferenceDisplay extends React.Component {
     } = this.props;
     return (
       <TableRow>
-        <TableData style={{overflowWrap: 'break-word'}}>
+        <TableData overflowWrap="break-word">
           {preference.nvt.name}
         </TableData>
-        <TableData style={{overflowWrap: 'break-word'}}>
+        <TableData overflowWrap="break-word">
           {preference.name}
         </TableData>
-        <TableData style={{overflowWrap: 'break-word'}}>
+        <TableData overflowWrap="break-word">
```

Comment:
Oh sorry you did misunderstood my comment. Should have been
```jsx
const StyledTableData = styled(TableData)`
  overflow-wrap: break-word;
`;

...

<StyledTableData>
  {preference.nvt.name}
</StyledTableData>
<StyledTableData>
  {preference.name}
</StyledTableData>
<TableData>
...
```
Hint: If this doesn't work currently TableData has to forwards the className prop.